sockeye/encoder.py

```diff
@@ -413,3 +453,171 @@ def get_rnn_cells(self) -> List[mx.rnn.BaseRNNCell]:
         Returns a list of RNNCells used by this encoder.
         """
         return self.forward_rnn.get_rnn_cells() + self.reverse_rnn.get_rnn_cells()
+
+
+class ConvolutionalEmbeddingEncoder(Encoder):
```

Comment:
would it be possible to write a simple unit test for this layer? I.e. provide random inputs, bind the symbol after this encoder and check for output shapes given input shapes?